src/sentry/api/endpoints/organization_metrics.py

```diff
@@ -20,35 +20,37 @@ def get_datasource(request):
     return MockDataSource()
 
 
-class ProjectMetricsEndpoint(ProjectEndpoint):
+class OrganizationMetricsEndpoint(OrganizationEndpoint):
     """Get metric name, available operations and the metric unit"""
 
-    def get(self, request, project):
+    def get(self, request, organization):
 
-        if not features.has("organizations:metrics", project.organization, actor=request.user):
+        if not features.has("organizations:metrics", organization, actor=request.user):
             return Response(status=404)
 
-        metrics = get_datasource(request).get_metrics(project)
+        projects = self.get_projects(request, organization)
```

Comment:
if there's all projects selected, can we find a way to avoid passing any filter to snuba? you're just going to run into query size limits this way, for large organizations

Reply:
`project_id` is currently a required field for snuba queries to the metrics dataset, so we'd have to change it there, too. That said, I agree that would make sense.